src/eos-metrics-instrumentation.c

```diff
@@ -226,11 +226,9 @@ systemd_dbus_proxy_new (void)
  * FALSE otherwise, in which case its contents should be ignored.
  */
 static gboolean
-get_user_id (GVariant *session_parameters, guint32 *user_id)
+get_user_id (const char *session_path, guint32 *user_id)
 {
     GError *error = NULL;
-    gchar *session_path;
```

Comment:
This variable is still referenced in the call to `g_dbus_proxy_new_for_bus_sync` below.